WooCommerce/src/main/kotlin/com/woocommerce/android/ui/blaze/campaigs/BlazeCampaignItem.kt

```diff
@@ -70,25 +70,23 @@ fun BlazeCampaignItem(
                     style = MaterialTheme.typography.subtitle1,
                     fontWeight = FontWeight.Bold,
                 )
-                Row(
-                    modifier = Modifier.padding(top = dimensionResource(id = R.dimen.major_100))
+                FlowRow(
+                    modifier = Modifier
+                        .padding(top = dimensionResource(id = R.dimen.major_100))
+                        .fillMaxWidth(),
```

Comment:
Minor np. If we replace this for `wrapContentWidth()` we'll keep the prior design on landscape mode. 

![Screenshot 2025-01-14 at 16 36 16](https://github.com/user-attachments/assets/b583532d-bbf8-464c-afaf-17d8f863c376)
![Screenshot 2025-01-14 at 16 36 00](https://github.com/user-attachments/assets/0202a791-4d0d-4b76-9350-3df6f4fe6cfd)

```suggestion
                        .wrapContentWidth(),
```

Reply:
Nice suggestion, thanks! TIL about `.wrapContentWidth()`. Works good in my check.